ApiBundle/Tests/Transformer/NodeTransformerTest.php

```diff
@@ -134,13 +134,23 @@ public function testTransformTransverse()
         $facade = $this->nodeTransformer->transform($this->node);
 
         $this->assertInstanceOf('OpenOrchestra\ApiBundle\Facade\NodeFacade', $facade);
-        $this->assertArrayHasKey('_self_form', $facade->getLinks());
-        $this->assertArrayNotHasKey('_self_duplicate', $facade->getLinks());
-        $this->assertArrayNotHasKey('_self_version', $facade->getLinks());
-        $this->assertArrayNotHasKey('_self_status_change', $facade->getLinks());
-        $this->assertArrayHasKey('_language_list', $facade->getLinks());
-        $this->assertArrayHasKey('_block_list', $facade->getLinks());
-        Phake::verify($this->router, Phake::times(5))->generate(Phake::anyParameters());
+        if (array_key_exists('_self_form', $facade->getLinks())) {
```

Comment:
well, i'm blocked then. I do not know how i should do that. Why _self_form could not be a condition of my test ?